apps/frontend/src/main.tsx

```diff
@@ -2,6 +2,8 @@ import { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
 import { createBrowserRouter, RouterProvider } from 'react-router-dom';
 
+import 'modern-normalize';
```

Comment:
Import the "@import 'node_modules/modern-normalize/modern-normalize.css';" in the index.css